### PR TITLE
solving lint issues.

### PR DESCRIPTION
**Pull request by niravasher**
Same as that of #3439 . Closed #3439 due to confusions in git.
image

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/

### CODE REVIEW

File: src/content/concepts/entry-points.md [hunk 1]
```diff
@@ -68,6 +68,41 @@ Below is a list of entry configurations and their real-world use cases:
...
+  }
+};
+```
+
+__webpack.prod.js__
+
+```javascript
+module.exports = {
+  output: {
+    filename: [name].contentHash.bundle.js
```
Comment (montogeek) [inline src/content/concepts/entry-points.md]: Please use valid JavaScript.

File: src/content/concepts/entry-points.md [hunk 2]
```diff
@@ -68,6 +68,41 @@ Below is a list of entry configurations and their real-world use cases:
...
+module.exports = {
+  entry: {
+    filename: [name].bundle.js
+  }
+};
+```
+
+__What does this do?__ We are telling webpack that we would like 2 separate entry points (like the above example).
+
+__Why?__ With this you can import required libraries or files that aren't modified (ex bootstrap, jquery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content Hash remains the same, which allows the browser to cache separately thereby reducing load time.
```
Comment (montogeek) [inline src/content/concepts/entry-points.md]:
```suggestion
__Why?__ With this you can import required libraries or files that aren't modified (i.e. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```

File: src/content/concepts/entry-points.md (hunk 1 above)
Comment (niravasher) [reply src/content/concepts/entry-points.md]: filename: [name].[contentHash].bundle.js
This fails travis CI test. @montogeek

File: src/content/concepts/entry-points.md (hunk 2 above)
Comment (EugeneHlushko) [reply src/content/concepts/entry-points.md]: @montogeek did you mean `e.g.` and not `i.e.` there, bootsrap and jquery are given as an example and not describing the statement in other words

File: src/content/concepts/entry-points.md (hunk 1 above)
Comment (EugeneHlushko) [inline src/content/concepts/entry-points.md]: It lacks single quotes no?
```suggestion
    filename: '[name].contentHash.bundle.js'
```

File: src/content/concepts/entry-points.md [hunk 3]
```diff
@@ -68,6 +68,41 @@ Below is a list of entry configurations and their real-world use cases:
...
+module.exports = {
+  entry: {
+    filename: [name].bundle.js
+  }
+};
+```
+
+__What does this do?__ We are telling webpack that we would like 2 separate entry points (like the above example).
+
+__Why?__ With this you can import required libraries or files that aren't modified (i.e. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```
Comment (EugeneHlushko) [inline src/content/concepts/entry-points.md:104]:
```suggestion
__Why?__ With this you can import required libraries or files that aren't modified (e.g. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```

File: src/content/concepts/entry-points.md [hunk 4]
```diff
@@ -68,6 +68,41 @@ Below is a list of entry configurations and their real-world use cases:
...
+  }
+};
+```
+
+__webpack.dev.js__
+
+```javascript
+module.exports = {
+  entry: {
+    filename: [name].bundle.js
```
Comment (EugeneHlushko) [inline src/content/concepts/entry-points.md:97]:
```suggestion
    filename: '[name].bundle.js'
```

File: src/content/concepts/entry-points.md [hunk 5]
```diff
@@ -68,6 +68,41 @@ Below is a list of entry configurations and their real-world use cases:
...
+  }
+};
+```
+
+__webpack.prod.js__
+
+```javascript
+module.exports = {
+  output: {
+    filename: '[name].contentHash.bundle.js'
```
Comment (EugeneHlushko) [inline src/content/concepts/entry-points.md:87]:
```suggestion
    filename: '[name].[contentHash].bundle.js'
```

File: src/content/concepts/entry-points.md [hunk 6]
```diff
@@ -68,6 +68,41 @@ Below is a list of entry configurations and their real-world use cases:
...
+module.exports = {
+  entry: {
+    filename: '[name].bundle.js'
+  }
+};
+```
+
+__What does this do?__ We are telling webpack that we would like 2 separate entry points (like the above example).
+
+__Why?__ With this you can import required libraries or files that aren't modified (e.g. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```
Comment (EugeneHlushko) [inline src/content/concepts/entry-points.md:104]: Is it HTML tho @montogeek WDYT?
```suggestion
__Why?__ With this you can import required libraries or files that aren't modified (e.g. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own chunk. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```

Comment (montogeek) [reply src/content/concepts/entry-points.md:104]: Nope

**Comment by montogeek** [comment]
@EugeneHlushko Do you want to finish this PR ourselves?

File: src/content/concepts/entry-points.md (hunk 6 above)
Comment (EugeneHlushko) [reply src/content/concepts/entry-points.md:104]: So what is it, i thoughts its chunking or?

**Comment by EugeneHlushko** [comment]
Yes since we can apply suggestions no need to chase the author

File: src/content/concepts/entry-points.md (hunk 6 above)
Comment (montogeek) [reply src/content/concepts/entry-points.md:104]: chunk is fine

**Comment by montogeek** [comment]
Alright, maybe they will answer :D

**Comment by EugeneHlushko** [comment]
All right lets see travis passing and merge

**Comment by montogeek** [comment]
Just realised that https://github.com/webpack/webpack.js.org/pull/3684 is very similar to this one.

**Comment by EugeneHlushko** [comment]
@montogeek its the way contributors tried to copy this work and apply suggestions adding another PR for same thing. Which is redundant now since we can commit suggestions on forks (finally)

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**